examples/gnrc_border_router/Makefile.esp.conf

```diff
@@ -0,0 +1,5 @@
+# Set the SSID and password of your WiFi network here
+CFLAGS += -DESP_WIFI_SSID=\"Your_WiFi_name\"
+CFLAGS += -DESP_WIFI_PASS=\"Your_secure_password\"
```

Comment:
With unifying I meant (and as I said this is unrelated and should be tackled in a separate PR): Have these configuration variables be generic and not dependent on ESP.

Reply:
Is there another board or driver that provides WiFi? AFAIK not at the moment.